src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big-js';
+
+/**
+ * Perform a simple binary math operation
+ * like the ones you found in a Basic Calculator
+ * @param {String|Number} numberOne First Operand
+ * @param {String} operation Valid operator for a Basic Calculator
+ * @param {String|Number} numberTwo Second Operand
+ * @returns {String} Numeric answer as String (because of `big-js` constrain)
+ */
+const operate = (numberOne, operation, numberTwo) => {
+  if (!['+', '-', 'X', '÷'].includes(operation)) {
+    return numberOne.toString();
+  }
+  let operand = numberOne;
+  if (numberOne.toString().match('^\\.')) {
+    operand = `0${numberOne}`;
+  }
+  operand = Big(operand);
+  return {
+    '÷': numberTwo === '0' || numberTwo === 0 ? '0'
```

Comment:
- why wrapped just `numberOne ` inside `big()`, u needed to also wrap `numberTwo`
- also concerning division, if `numberTwo == 0` we get INFINITY, not zero normally (you could also display an error if you like, something like `"division by ZERO error"`

Reply:
Thanks for your review! 

> *why wrapped just `numberOne` inside `big()`, u needed to also wrap `numberTwo`

About this I followed the documentation:
![image](https://user-images.githubusercontent.com/36519478/76103834-5ee9ad00-5fa0-11ea-8503-d57cbf051205.png)

So it seems like the second number is optional to be inside `big()`. I would test if wrapping `numberTwo` as work as well!

> - also concerning division, if numberTwo == 0 we get INFINITY, not zero normally (you could also display an error if you like, something like "division by ZERO error"

I'll take that in care on that, thanks for the advice!